test/test_ripper_tags.rb

```diff
@@ -95,7 +95,7 @@ module A::B
     assert_equal 'A::B::D', tags[2][:class]
   end
 
-  def test_extract_access
+  def test_extract_access_and_kind
```

Comment:
Changes made, thank you for reviewing. (got a busy weekend, sorry for the delay)